### ISSUE TITLE
Question: Balance of the Quartz Canyon

### ISSUE THREAD

**Issue by saltyseadoggo** [TerraformersMC/Cinderscapes#88]
I wanted to open a discussion with regards to the balance of the Quartz Canyon biome. This biome presents the player with very large quantities of quartz in different formes and varieties, all of which can be used to craft redstone components, namely observers, which require them. In vanilla Minecraft, Mojang has kept a fairly tight reign on how much quartz players can obtain easily: it is only so common in the vanilla biomes it appears in, and can only be farmed via a zombified Piglin farm in conjunction with a Piglin trading setup, which require quite a bit of setup, as well as player intervention in some setups to compress golden nuggets or handle barter drops.
The quartz canyon biome makes obtaining large quantities of redstone-viable quartz much, much easier by spawning quartz ores in four varieties _very_ frequently, as well as by generating crystalline and polypite quartz crystals. One can fill one's inventory with quartz within less than twenty minutes simply by harvesting all of the ores that are visible on the ground with Fortune III, to say nothing of tearing down the crystals, harvesting the polypites or deliberately mining underground. I am personally concerned that this is potentially unbalanced, as it is much harder to obtain quartz for redstone components in vanilla Minecraft by Mojang's design.
As a side note, harvesting all of this quartz is an easy and low-effort way of gathering experience levels in fair quantities, much faster than mining quartz in vanilla or in other Cinderscapes biomes. By simply mining the ores in the ground, one can typically go from level 0 to 30 in less than ten minutes. I see this as a lesser concern than the much easier gathering of redstone components, but it's a byproduct nonetheless.

I wanted to open this up for discussion, as I may be missing another side to this as it might pertain to the argument for easier obtainable quartz in vanilla. If you deem this to indeed be a reasonable issue, potential solutions could include decreasing the quantities of quartz ores in these biomes, as their current spawn rates are a bit ridiculous; making the large quartz crystals only sometimes drop quartz items, as if they sometimes just shatter entirely, like eyes of ender; and/or perhaps letting more dangerous mobs like Zoglins spawn here, assuming you can resolve #48 , to increase the risk of gathering quartz in these biomes.~

**Comment by ghost** [comment]
I agree there probably needs to be more risk associated with a location full of so much value. In the most recent update I turned down the lighting in that biome by turning off the ability for the polypite quartz to emit light. This should make mob spawns a little more common to hopefully balance it a little bit.

When we figure out Smolder I think it would make sense to make this biome significantly rarer just because of the nature of what's in it.

**Comment by Angular-Angel** [comment]
It should also be significantly smaller, if possible. Maybe cut it down to just like 3-5 chunks?

**Comment by dranorter** [comment]
Normally I encounter more quartz than I need if I mine for Netherite, or if I simply decide to set up tunnels going somewhere. However, recently I needed quartz for a building project, and decided to find a quartz canyon. The first one I encountered in my current world was very small, a peninsula of netherrack hanging high above lava, ghasts and hoglins everywhere. Basically it seemed like the quartz canyon biome had hit the edge and underside of the peninsula, and the crimson forest claimed the top. Which looked fantastic; these crystal structures look great jutting out of other biomes. But more importantly, it was definitely risky getting up there and I ultimately decided it was too risky to mine for long.

The next one I found was much larger, but similarly very airy, hanging above lava.

Basically I think making this biome small works great, since it looks so good mixing in with other things.  And when it is bigger, I think a fairly effective way of making it dangerous is if it would tend to be small amounts of netherrack suspended above lava. Having the biome be more open makes it fit the word "canyon" better anyway. In my first world using Cinderscapes, the first quartz canyon I found was very dense, providing both ridiculous quantities of ore, and plenty of places to tunnel for safety. I'd say as long as that doesn't happen, I'm happy with the balance.

I wouldn't necessarily want the biome to be small every single time; it does look nice larger too, So I'd be in favor of also finding other ways to make it more dangerous.

**Comment by gniftygnome** [comment]
> In the most recent update I turned down the lighting in that biome by turning off the ability for the polypite quartz to emit light.

As of today's releases, this is now configurable (default of luminance 4, unchanged from Vaerian's update).  Note that changing this setting in an existing world will not cause existing lighting caches to be updated.  You must restart Minecraft and even then existing chunks will not become brighter unless something causes a lighting cache update.
